CHANGELOG.md

```diff
@@ -18,8 +18,15 @@ Main (unreleased)
 
 - Rename component `prometheus.exporter.agent` to `prometheus.exporter.self` to clear up ambiguity. (@hainenber)
 
+### Deprecations
+
+- Classic modules have been deprecated and will be removed in v1.0. (@wildum)
```

Comment:
Classic modules is a new term being introduced here, so readers might not be able to follow this. Can we say this instead?

> `module` components have been deprecated in favor of `import` and `declare` configuration blocks. These deprecated components will be removed in the next release. (@wildum) 

(I would avoid saying v1.0 explicitly because we don't know with 100% certainty if 0.41 == 1.0, even if that's the plan)